manifests/01-credentials-request-azure.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: openshift-image-registry-azure
+  namespace: openshift-cloud-credential-operator
```

Comment:
1. yes, we have manifests for AWS and Azure, do CVO will create credential requests for both providers
2. my understanding is that it will ignore requests for platforms other than current: https://github.com/openshift/cluster-image-registry-operator/pull/290#discussion_r290345417